packages/powercontrol/png-icons.nix

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenv,
+  pkgs,
+  ...
+}: let
+  shutdownIconName = "system-shutdown-symbolic";
+  rebootIconName = "system-reboot-symbolic";
+
+  iconColour = "white";
+
+  changeColourCcs = "path { fill: ${iconColour} !important; }";
```

Comment:
The most minor nitpick of all times, this should be changeColorCss. Colour is old British English

Reply:
Is this colour thing even needed? Why there's some much stuff in this derivation when it seems only 2 of the images are being used?

Reply:
Why is it exactly old British English? It's just UK vs US variants even nowadays.
https://www.scribbr.com/frequently-asked-questions/color-colour/

Nix itself uses UK English -- so I try to be consistent with it.

Reply:
What do you mean by "some much stuff"? What is the point?

Reply:
I have changed to 'color' everywhere